internal/pkg/log/log.go

```diff
@@ -63,7 +64,7 @@ type contextLogger interface {
 	Sync() error
 }
 
-type withFields interface {
-	With(args ...any) Logger
+type withAttributes interface {
+	With(attrs ...attribute.KeyValue) Logger
 	WithComponent(component string) Logger
 }
```

Comment:
I prefer a one ways how to work with attributes in logger, tracer and `ctxattrs`, it was discussed, but no final decision was made.

If we specify the data type, we can refactor it more easily in the future to a different logger than Zap.

Reply:
refactor to `log/slog`?

Reply:
yes, we discussed the slog with Jachym, but we didn't want to complicate the logs refactoring now